Flow.Launcher.Infrastructure/UserSettings/DataLocation.cs

```diff
@@ -30,5 +30,7 @@ public static bool PortableDataLocationInUse()
         }
 
         public static readonly string PluginsDirectory = Path.Combine(DataDirectory(), Constant.Plugins);
+        public static readonly string SettingsDirectory = Path.Combine(DataDirectory(), "Settings");
+        public static readonly string PluginSettingsDirectory = Path.Combine(SettingsDirectory, Constant.Plugins);
```

Comment:
`PluginSettingsDirectory` can refactored to `Path.Combine(DataDirectory(), "Settings", Constant.Plugins);` to remove `SettingsDirectory`